src/components/ProfilePage/AddHouseForm/AddHouseForm.tsx

```diff
@@ -71,7 +75,32 @@ function AddHouseForm() {
   const removeImage = (name: string) => setImages((prevState) => prevState?.filter((img) => img.name !== name));
 
   const handleSend = (fields: AddHouseFormFields) => {
-    sendHouseDataWithPhotos(images, fields);
+    const payload = new FormData();
+
+    images.forEach((image) => payload.append('images[]', image, image.name));
+
+    const additionalDetails = fetchedAddressOptions?.data?.data?.[0] || {};
+
+    const extendedFields = {
+      ...fields,
+      owner: '623c418ae3d719b1b508d835',
+      country: additionalDetails.country as string,
+      lat: additionalDetails.latitude as number,
+      lng: additionalDetails.longitude as number,
```

Comment:
Some keys aren't mapped to what they are named like on the back-end. I'm guessing you'd want to change the names on the back-end, right?